Dockerfile

```diff
@@ -11,6 +11,9 @@ RUN go build -ldflags="-X github.com/prometheus/common/version.Version=${caexpor
 FROM alpine:3.16
 LABEL maintainer "Bringg DevOps <devops@bringg.com>"
 
+RUN apk --no-cache -X https://dl-cdn.alpinelinux.org/alpine/edge/testing add gosu
```

Comment:
Not sure you need `gosu` here, it's a fairly simple image so you can just use `USER nobody` directive